app/ui/src/app/integration/import-export/import/integration-import.component.ts

```diff
@@ -18,23 +18,30 @@ import {
 
 @Component({
   selector: 'syndesis-import-integration-component',
-  templateUrl: './integration-import.component.html'
+  templateUrl: './integration-import.component.html',
+  styleUrls: ['./integration-import.component.scss']
 })
 export class IntegrationImportComponent implements OnInit {
   public uploader: FileUploader;
+  public hasBaseDropZoneOver = false;
 
   constructor(
     public notificationService: NotificationService,
     private integrationSupportService: IntegrationSupportService,
   ) {}
 
+  public fileOverBase(e) {
+    this.hasBaseDropZoneOver = e;
+  }
+
```

Comment:
TypeScript class members are `public` by default so we should not include such accessor in our code. Feel free to remove it not only from your code but from any other instance of code you find on the way.